src/responseStatuses.js

```diff
@@ -0,0 +1,79 @@
+export const responseStatuses = {
```

Comment:
Change the file to `.ts` and:

```suggestion
interface ResponseStatusInfo {
  code: number;
  name: string;
  description: string;
}

export const responseStatuses: Record<number, ResponseStatusInfo>  = {
```